api/src/main/java/org/pmiops/workbench/db/model/User.java

```diff
@@ -130,4 +131,13 @@ public void setFirstSignInTime(Timestamp firstSignInTime) {
   public void setAuthorities(Set<Authority> newAuthorities) {
     this.authorities = newAuthorities;
   }
+
+  @OneToMany(mappedBy = "user")
+  public Set<WorkspaceUserRole> getWorkspaceUserRoles() {
+    return workspaces;
+  }
+
+  public void setWorkspaceUserRoles(Set<WorkspaceUserRole> userRoles) {
```

Comment:
I can't remember if we've done this in the past -- a setter for a mapped field. A while back I ran into a bug where simple overwiting did *add* the new rows, but did not *delete* the old rows. So if you haven't, that'd be good to stick in a unit test. (The solution in that case was to clear the list and add new items, instead of overwriting the list.)